conda_recipes/maya-vray-2025/README.md

```diff
@@ -0,0 +1,22 @@
+# V-Ray conda build recipe
+Use this recipe to create a V-Ray for Maya conda package to use with AWS Deadline Cloud. Conda packages let you customize the software you can use with your Deadline Cloud deployment. Read more about how to host a conda channel for these custom conda packages below.
```

Comment:
Which specific content is this "Read more" sentence describing?

Reply:
The intention is to say something along the line of read more about how to make this work below. Yes it seem like this read more is not fitting here. I will rephrase it :)